src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -176,9 +176,20 @@ private boolean isVisibilitySetToAnonymousToRecipientAndInstructors(FeedbackQues
     }
 
     private boolean isVisibilitySetToAnonymousToRecipientVisibleToInstructors(FeedbackQuestionAttributes question) {
-        boolean responsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
-                && question.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS)
-                && question.showResponsesTo.contains(FeedbackParticipantType.RECEIVER);
+        boolean responsesVisibleOnlyToRecipientAndInstructors;
+
+        if (question.questionType.equals(FeedbackQuestionType.CONTRIB)) {
```

Comment:
I feel it should be a common option for better user experience. Maybe I should make a new option `Shown anonymously to recipient and team members, visible to instructors`?